File: Week1/mario-more/mario.c
#include <stdio.h>
#include <cs50.h>

int main(void)
{
    int n;
    do
    {
        n = get_int("Width: ");
    }
    while (n < 1 || n > 8) ;
    for(int i=0; i<n; i++)
    {
        for(int j=1; j<n-i; j++)
        {
            printf(" ");
        }

        for(int k=0; k<=i; k++)
        {
            printf("#");
        }
        printf("  ");
        for(int l=0; l<=i; l++)
        {
            printf("#");
        }
    printf("\n");
    }

}

File: Week6/sentimental-mario-less/mario.py
from cs50 import get_int

while True:
    h = get_int("Height: ")
    if h < 9 and h > 0:
        break

for i in range(h):
    for j in range(h-i-1):
        print(" ", end = "")
    for k in range(i+1):
        print("#", end = "" )
    print()
